jjanmo/src/main/kotlin/lotto/Main.kt

```diff
@@ -1,5 +1,88 @@
 package lotto
 
+import java.time.DayOfWeek
+import java.time.Duration
+import java.time.LocalDate
+import java.time.LocalDateTime
+import kotlin.random.Random
+
+class LottoDayCalculator {
+    fun generateLottoNumber(): List<Int> {
+        val lotto = mutableListOf<Int>()
+        while (lotto.size < 6) {
+            val newNumber = Random.nextInt(1, 45)
```

Comment:
```suggestion
val newNumber = Random.nextInt(1, 46)
```
이렇게 해야 1 ~ 45까지 뽑는다고 한다. ( 끝자리 + 1 )